src/handlers/activities.js

```diff
@@ -73,7 +76,8 @@ class ActivitiesHandler extends BaseHandler {
       details: {
         licenceNumber: cache.licenceNumber,
         postcode: cache.postcode,
-        year: cache.year
+        year: cache.year,
+        maxDaysFished
```

Comment:
No problem with this bit per se, but above where you're generating the rivers it is too difficult to understand . I couldn't tell you what `![].concat(...activities.map(a => a.river)` is doing. You've put a comment above to try and explain, but breaking up your terse expression into several lines with well named variables could be much easier to comprehend. It will probably help with reducing the number of parameters used by the function too, which as I've said is a problem.